twtxt/cli.py

```diff
@@ -68,6 +69,14 @@ def cli(ctx, config, verbose):
 @click.pass_context
 def tweet(ctx, created_at, twtfile, text):
     """Append a new tweet to your twtxt file."""
+
+    """spell check the tweet before sending, and let the user know!"""
+    subprocessReturn = subprocess.check_output(["aspell", "-a"], input=text, universal_newlines=True)
+    if subprocessReturn.find("&") != -1:
+        click.echo("Theres a spelling mistake in your tweet:\n"+text+"\n")
```

Comment:
Always place spaces between operators like `+` and prefer `str.format` instead of `+`